pytest_plugins/markers.py

```diff
@@ -1,3 +1,6 @@
+# Custom markers for robottelo tests
```

Comment:
Worth removing them from `robottelo.decorators` ?

Reply:
Not sure exactly what you mean removing here - there are plenty of pytest marker re-naming I would like to remove from `decorators.__init__` and update all test modules to import and use them directly from pytest.  I don't think it falls into the scope of this PR.

This change is just actually registering the existing markers with pytest so we don't get warnings about unregistered markers.

Reply:
The same markers defined in your `pytest_plugins/markers.py` are defined in `robottelo.decorators.__init__.py`. So I am talking abt the same that u mentioned above comment.

I am ok to improve it from another PR.

Reply:
Gotcha. I didn't want to add `robottelo.decorators.__init__` to conftest, but rather register them with pytest from a new markers module, and move marker definitions into this module.